layers/osx/README.org

```diff
@@ -32,11 +32,7 @@ and we encourage you to do so :)
 
 ** Layer
 
-To use this configuration layer, add it to your =~/.spacemacs=
-
-#+BEGIN_SRC emacs-lisp
-  (setq-default dotspacemacs-configuration-layers '(osx))
-#+END_SRC
+To use this configuration layer, add it to your =~/.spacemacs=. You will need to add `osx` to the existing list of `dotspacemacs-configuration-layers` in this file.
```

Comment:
This has to be solved in a more general way (i.e. in all layers) as discussed in  #2113